aten/src/ATen/cuda/tunable/Tunable.cpp

```diff
@@ -294,10 +294,14 @@ TuningContext::TuningContext() :
     enable_{false},
     tuning_enable_{true},
     manager_initialized_{false},
+    write_file_on_exit_{true},
+    numerics_check_enable_{true},
     max_tuning_duration_ms_{30},
     max_tuning_iterations_{100},
     max_warmup_duration_ms_{0},
     max_warmup_iterations_{0},
+    icache_flush_{false},
```

Comment:
hmm, if we believe icache flush and rotating buffer are useful, i think we should use a better initial value?